### PR TITLE
Added polling.method option

### PR DESCRIPTION
**Pull request by codejamninja**
Fixes issue #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,16 +50,17 @@ Check out [chris.bolin.co/offline](https://chris.bolin.co/offline) for a simple
...
+| ------------------ | ----------- | ---------------------------------         | ------------------------------                                                    |
+| `polling`          | Obj or Bool | Config for polling fallback [1]           | [see below]                                                                       |
+| `polling.enabled`  | Boolean     | Force polling on or off                   | Depends on the browser [1]                                                        |
+| `polling.url`      | String      | URL to pool for connection status         | `"https://ipv4.icanhazip.com"`                                                    |
+| `polling.interval` | Number      | How often (in ms) to poll                 | `5000`                                                                            |
+| `polling.timeout`  | Number      | How long (in ms) before timeout           | `5000`                                                                            |
+| `polling.method`   | String      | method used to poll for connection status | `CONNECT`, `DELETE`, `GET`, `HEAD`, `OPTIONS`, `PATCH`, `POST`, `PUT`, or `TRACE` |
```
Comment (chrisbolin) [inline README.md]: The last column shows the Default value, so it should just be `HEAD`

And the Description should be `method used to poll for connection status` -> `method to pool for connection status`

Comment (chrisbolin) [reply README.md]: and why are all the other rows changed? just a editor/prettier auto change?

Comment (codejamninja) [reply README.md]: The same reason all the columns were lined up before I made the changes.

Comment (codejamninja) [reply README.md]: Yes, the code formatter is called emacs :)

https://www.gnu.org/software/emacs

Comment (chrisbolin) [reply README.md]: are you going to make the requested changes or just be snarky?

**Comment by chrisbolin** [comment]
_closed due to inactivity_

**Pull request closed without merging**